app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -17,6 +18,44 @@
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs .govuk-breadcrumbs__list-item {
+  margin-left: 0;
+  padding-left: 0;
+  margin-right: 20px;
```

Comment:
Why are we overriding so much of `govuk-frontend` here? Their breadcrumbs seem to work perfectly well.

Reply:
Overrides are to facilitate touch target and chevron origin switching from "before" to "after"